numpy/distutils/system_info.py

```diff
@@ -1131,6 +1133,7 @@ def calc_info(self):
         info = {}
         atlas_libs = self.get_libs('atlas_libs',
                                    self._lib_names + self._lib_atlas)
+        atlas_libs = self.get_libs('libraries', atlas_libs)
         lapack_libs = self.get_libs('lapack_libs', self._lib_lapack)
```

Comment:
Why is `lapack_libs` still used as the `x_libs` format, but `atlas_libs` aliased to `libraries`?

Reply:
Yes, this is a bit problematic with ATLAS. ATLAS can provide either BLAS or BLAS+LAPACK(sub) or BLAS+LAPACK, with the middle option being only a subset of routines which are optimized.
I would suggest *any* user to put all libraries (both BLAS and LAPACK) in a single library entry, however it seems that the dev of the `atlas_info` class recommended LAPACK and BLAS libraries to be split. The only case where this would be troublesome is to use ATLAS BLAS but another LAPACK lib.

Basically I think this is why there is an abstraction of another key for only the LAPACK libraries.

So the final atlas section would look like:
```bash
[atlas]
libraries = f77blas,cblas,atlas
lapack_libs = lapack_atlas
```
And then a user could do `get_info('blas_opt').update(get_info('lapack_mkl'))` for instance (dummy example ;) )

Reply:
I think splits are possible, they're different libraries (e.g. BLIS + Flame).

> ATLAS can provide either BLAS or BLAS+LAPACK(sub) or BLAS+LAPACK, with the middle option being only a subset of routines which are optimized.

I have never seen ATLAS used with numpy/scipy as anything other than "all of BLAS + LAPACK".

Reply:
@rgommers agreed, me neither. But the possibility *is* there.